src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java

```diff
@@ -483,7 +483,7 @@ private void doKafkaDataContinuation(
         private static final int UNATTACHED = -1;
 
         private final MessageConsumer applicationThrottle;
-        private final long applicationId;
+        private long applicationId;
```

Comment:
Why is `applicationId` non-final now?

Reply:
Must've been for a change I later backed out. Fixed.